en/docs/develop/customizations/customizing-the-developer-portal/styling-the-logo-and-header.md

```diff
@@ -25,7 +25,7 @@ const Configurations = {
 #### Example
 
 We can change the logo and header background as follows by changing the above parameters.
```

Comment:
above parameters or below parameters?

Reply:
Why do we have a configuration on top?

Reply:
What is the file we need to change? Where can we find it? Shall we add these steps in the doc too?

Reply:
Updated in the following commits